run_tests.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/**
+ * This script runs tests for all packages in `./packages` and
+ * example projects in `./examples`.
+ */
+
+const fs = require('graceful-fs');
+const path = require('path');
+const chalk = require('chalk');
+const execSync = require('child_process').execSync;
+
+const PACKAGES_DIR = './packages';
+const EXAMPLES_DIR = './examples';
+
+const packages = fs.readdirSync(PACKAGES_DIR)
+  .map(file => path.resolve(PACKAGES_DIR, file))
+  .filter(f => fs.lstatSync(path.resolve(f)).isDirectory());
+
+const examples = fs.readdirSync(EXAMPLES_DIR)
+  .map(file => path.resolve(EXAMPLES_DIR, file))
+  .filter(f => fs.lstatSync(path.resolve(f)).isDirectory());
+
+/**
+ * Run a command or a list of commands synchronously
+ * @param {String,Array<String>} commands
+ * @example runCommand(['npm i', 'npm test'], process.cwd());
```

Comment:
do you mind removing this docblock? I think the API for `runCommands` is self-explanatory. The problem with these kinds of comments is that they easily get out of sync.